app/topics/topic-management/topic-input/topic-input.component.html

```diff
@@ -1,23 +1,36 @@
 <md-card class="default-card topic-card" [class.subtopic]="depth > 0">
   <md-card-title>
     <md-input placeholder="{{'topic title' | translate}}" #title="ngModel" required
-              [(ngModel)]="topic.title" name="title"
-              dividerColor="title.touched && title.invalid ? 'warn' : 'primary'"
-    >
+              [(ngModel)]="topic.title" name="title" (keyup)="getQueryTopic()"
+              dividerColor="title.touched && title.invalid ? 'warn' : 'primary'">
       <md-hint [hidden]="title.valid || title.untouched" class="error-hint">
         {{'title required' | translate}}
       </md-hint>
     </md-input>
-    <md-radio-group [(ngModel)]="topic.status"
-                    #status="ngModel"
-                    ngDefaultControl>
+    <div class="item-wrapper" (if)="queriedTopics || topic.title.length >=3">
+      <div>
+        <ul class="dropdownlist dropdown-menu" aria-labelledby="dropdownNotifications">
+          <li *ngFor="let topiclist of queriedTopics">
```

Comment:
I would suggest to put some headline `similar topics` to the dropdown, so the user knows what these dropdown means.